opt/scopeserver/dataserver/utils/loom_file_handler.py

```diff
@@ -80,71 +57,47 @@ def change_loom_mode(
 
         else:
             logger.debug(f"Reopening file as r")
-            if keep_ss:
-                new_partial_md5_hash = LoomFileHandler.get_partial_md5_hash(abs_file_path, 10000)
-                logger.debug(
-                    f'Keeping old SS, moving {os.path.join(os.path.dirname(abs_file_path), partial_md5_hash + ".ss_pkl")} \
-                        to {os.path.join(os.path.dirname(abs_file_path), new_partial_md5_hash + ".ss_pkl")}'
-                )
-                os.rename(
-                    os.path.join(os.path.dirname(abs_file_path), partial_md5_hash + ".ss_pkl"),
-                    os.path.join(os.path.dirname(abs_file_path), new_partial_md5_hash + ".ss_pkl"),
-                )
             self.active_looms[abs_file_path] = self.get_loom(loom_file_path=loom_file_path)
             logger.info(f"{loom_file_path} now {self.active_looms[abs_file_path].get_connection().mode}")
 
-        logger.debug(f"Checking MD5")
-        new_partial_md5_hash = LoomFileHandler.get_partial_md5_hash(abs_file_path, 10000)
-        logger.debug(f"Old MD5 is: {partial_md5_hash} New MD5 is: {new_partial_md5_hash}")
-
-        if (partial_md5_hash != new_partial_md5_hash) and not keep_ss:
-            try:
-                logger.debug("Removing old SS")
-                os.remove(os.path.join(os.path.dirname(abs_file_path), partial_md5_hash + ".ss_pkl"))
-            except Exception as e:
-                logger.debug("Couldn't remove pickle SS")
-                logger.debug(e)
-                pass
         return self.active_looms[abs_file_path].get_connection()
 
-    def get_loom_absolute_file_path(self, loom_file_path: str) -> str:
-        return os.path.join(self.loom_dir, loom_file_path)
+    def get_loom_absolute_file_path(self, loom_file_path: Path) -> Path:
+        return self.loom_dir / loom_file_path
```

Comment:
This function [isn't really necessary](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve)

Reply:
This function takes in loom file names which are not a full path object, which means we need to build the path still. Maybe `get_loom_absolute_file_path` isn't the best name

Reply:
To be clearer, the frontend often send just the name of a loom file, which we need to build the path from, based on the knowledge of where the backend is storing these files

Reply:
OK I see